package.json

```diff
@@ -28,6 +28,7 @@
     "@shopgate/pwa-core": "^0.1.3",
     "@shopgate/pwa-tracking": "^0.1.2",
     "@shopgate/pwa-webcheckout-shopify": "^0.1.4",
+    "@shopgate/react-hammerjs": "^0.5.2",
```

Comment:
Please remove it from here. It will come with pwa-common

Reply:
other way around like discussed ;)